src/main/java/wolox/training/models/Book.java

```diff
@@ -105,6 +117,7 @@ public String getYear() {
     }
 
     public void setYear(String year) {
+        Preconditions.checkNotNull(year, "Year cannot be null");
```

Comment:
In addition to the change suggested above, we could add a check that verifies that the content of this string is numeric

`.checkArgument (StringUtils.isNumeric (year), "error message");`

Reply:
Add new validate to set